front/src/components/HeaderMenu/HeaderMenuContainer.tsx

```diff
@@ -139,7 +139,7 @@ function HeaderMenuContainer() {
       },
       {
         text: "Coordinator Admin",
-        requiredAbp: [],
+        requiredAbp: ["manage_volunteer"],
```

Comment:
This needs an extra s: `manage_volunteers`
ABPs are listed [here](https://docs.google.com/spreadsheets/d/1W4YWcc59wUFUWgReumdH6DQ4zU7JcTgvf6WEbdqaGHQ/edit?gid=1124938013#gid=1124938013) (4th sheet).

Reply:
Thank you @pylipp !